.golangci.yml

```diff
@@ -13,17 +13,30 @@ linters:
     - misspell
     - unused
     - whitespace
+  disable:
```

Comment:
errcheck is a program for checking for unchecked errors in go programs. Disabling this adds flexibility in the code base by preventing every function that returns an error from being assigned to a variable or being checked

Reply:
This is bad and good based on usage. Disabling this will result in code not handling returned errors not being flagged. On the other hand, its common not to check for errors when using an api like `Fprintf`.

Reply:
That's true, I have been debating this and I'm not sure what we would prefer

Reply:
Based on an offline sync with @draychev we will attempt to enable errcheck in a separate PR